### PR TITLE
Chore: Update script names to latest conventions

### PR DESCRIPTION
**Pull request by jstoffan**
It'll probably help quite a bit to hide whitespace diffs on this changeset.

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @jstoffan has signed the CLA. Thanks for the pull request!

File: package.json [hunk 1]
```diff
@@ -1,148 +1,153 @@
...
+        "webpack-bundle-analyzer": "^2.9.1",
+        "webpack-dev-server": "^2.11.3"
+    },
+    "engines": {
+        "node": ">=8.9.4",
+        "yarn": ">=1.10.0"
+    },
+    "scripts": {
+        "build": "yarn setup && yarn clean && yarn build:i18n && yarn lint && yarn build:dev",
+        "build:ci": "yarn setup && yarn webpack --progress --colors --config build/webpack.config.js",
```
Comment (ConradJChan) [inline package.json]: Do we not need relative paths to webpack & karma anymore?

File: package.json [hunk 2]
```diff
@@ -1,148 +1,153 @@
...
+        "lint:css": "NODE_ENV=dev yarn stylelint 'src/lib/**/*.scss'",
+        "lint:js": "NODE_ENV=dev yarn eslint src/lib",
+        "release": "yarn setup && yarn clean && yarn build:i18n && yarn lint && yarn test && yarn build:prod",
+        "release:major": "./build/release.sh -m",
+        "release:minor": "./build/release.sh -n",
+        "release:patch": "./build/release.sh -p",
+        "setup": "yarn install",
+        "start": "yarn setup && yarn build:dev --watch",
+        "start:dev": "yarn build:i18n && LANGUAGE=en-US BABEL_ENV=dev NODE_ENV=dev yarn webpack-dev-server --config build/webpack.config.js",
+        "test": "yarn setup && yarn clean && yarn build:i18n && NODE_ENV=test yarn karma start build/karma.conf.js",
```
Comment (ConradJChan) [inline package.json]: Is the addition of `yarn build:i18n` needed for the tests?

File: package.json (hunk 1 above)
Comment (jstoffan) [reply package.json]: I don't think so. Prepending with `yarn` should force the commands to use local packages.

File: package.json (hunk 2 above)
Comment (jstoffan) [reply package.json]: Yes, unfortunately. The prior command had it but it was hidden away in the `ci` command, iirc.

**Pull request merged**